.github/workflows/build.yml

```diff
@@ -0,0 +1,117 @@
+name: Build Code
+
+on:
+  push:
+  pull_request:
+    types: [opened]
+
+env:
+  VMA_VULKAN_VERSION: "1.3.283.0"
+  VMA_VULKAN_SDK_PATH: "$GITHUB_WORKSPACE/../vulkan_sdk/"
+  VMA_VULKAN_SDK_CHECKSUM_LINUX: "8005e2cf3e89c80cbe1c0d0a259c88248de3257b4fc6fdefb47409edb3e43ecb"
```

Comment:
These checksums don't seem to be used. Can we get rid of them? Even if you could use them, I would rather avoid having to update such checksums every time we update Vulkan SDK.